### ISSUE TITLE
Deal with Sweetalert deprecation warning

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3498]
When doing an operation that causes a Sweetalert dialog get this warning in the browser
```
SweetAlert2: The parameter "useRejections" is deprecated and will be removed in the next major release.
```

Should probably deal with this and move `Alerts` into some global place where it can imported.

**Comment by aaronjudd** [comment]
Should be resolved in #3620

**Comment by zenweasel** [comment]
Still getting this deprecation warning
